llama_stack/providers/inline/agents/meta_reference/agent_instance.py

```diff
@@ -535,7 +535,7 @@ async def _run(
                     tool_call = message.tool_calls[0]
 
                     name = tool_call.tool_name
-                    if not isinstance(name, BuiltinTool):
+                    if not isinstance(name, BuiltinTool) or name not in enabled_tools:
```

Comment:
This check seems redundant? `not isinstance(name, BuiltinTool)`. Can we just check if the name is in enabled?

Reply:
@yanxi0830 - I'm assuming this is because custom tools aren't supported at the moment is that correct?

Reply:
I've just checked on main - yeah custom tools don't seem to be supported atm (even if they are enabled):

<img width="909" alt="image" src="https://github.com/user-attachments/assets/f22cca91-eb94-48e1-92b8-28aad534639c" />